package.json

```diff
@@ -57,6 +58,7 @@
         "eslint-plugin-security-node": "^1.1.4",
         "eslint-plugin-unicorn": "^54.0.0",
         "husky": "^9.0.7",
+        "lcov-result-merger": "^5.0.1",
```

Comment:
This might not be needed, codecov should automatically merge the results as long as we upload all the different lcov.info files. https://about.codecov.io/product/feature/report-merging/#:~:text=Codecov%20will%20merge%20the%20reports,or%20across%20different%20CI%2FCDs.

Reply:
I was just hoping we could just do a single call rather than multiple and do everything with globs so we wouldn't have to revisit this if a new package was added. It looks like the result merge might be mangling the results and codecov doesn't seem to like that though